scripts/capture.py

```diff
@@ -8,14 +8,29 @@
 FILE = '%H:%M:%S.png'
 
 
+def matches(frame, threshold):
+    grayimg = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
+
+    _, thresh = cv2.threshold(grayimg, threshold, 255, cv2.THRESH_BINARY)
+    nonzero = cv2.countNonZero(thresh)
+
+    return thresh.size - nonzero < thresh.size // 3
+
+
 @click.command()
 @click.option('--warmup', default=15,
               help='Warmup camera by capturing given number of frames')
 @click.option('--threshold', default=60,
               help='Threshold value for grayscale to binary image')
 @click.option('--path', default='.',
               help='Path where to save captured images')
-def capture_and_save(warmup, threshold, path):
+@click.option('--image', default='',
+              help='Specify path and get True/False for bright/dark image')
+def capture_and_save(warmup, threshold, path, image):
+    if image != '':
+        print(matches(cv2.imread(image, 1), threshold))
```

Comment:
@Adman Could we say something like 

```python
"Matches image {} with threshold {}: {}".format(image, threshold,  matches(cv2.imread(image, 1), threshold))
```

It is quite difficult to figure out what the defaults were for instance, without checking the code/help.